python/tk_multi_workfiles/file_save_form.py

```diff
@@ -134,18 +134,22 @@ def _do_init(self):
         self._ui.browser.enable_show_all_versions(False)
         self._ui.browser.enable_user_filtering(False)
         self._ui.browser.set_models(self._my_tasks_model, self._entity_models, self._file_model)
-        env = WorkArea(app.context)
         current_file = self._get_current_file()
         self._ui.browser.select_work_area(app.context)
         self._ui.browser.select_file(current_file, app.context)
 
-        # initialize the browser with the current file and environment:
-        self._on_browser_file_selected(current_file, env)
+        try:
+            env = WorkArea(app.context)
+        except TankError, e:
+            self._disable_save_and_warn(str(e))
```

Comment:
Actually, the try except is not necessary. I've added a comment why in the code.